<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtifactMarkupWidget</class>
 <widget class="QWidget" name="ArtifactMarkupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark-up Artifacts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
       <item>
        <widget class="QScrollArea" name="_videoScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>206</width>
            <height>453</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="_frameLabel">
           <property name="text">
            <string>Frame</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="CGTVideoControls" name="_videoControl" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="controlsLayout">
       <item>
        <layout class="QHBoxLayout" name="modeLayout">
         <item>
          <widget class="QRadioButton" name="_drawButton">
           <property name="text">
            <string>Draw</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">_markerButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="_moveButton">
           <property name="text">
            <string>Move </string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">_markerButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="_deleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">_markerButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="propsLayout">
         <item>
          <layout class="QHBoxLayout" name="regionsLayout">
           <item>
            <widget class="QComboBox" name="_regionsComboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Regions</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="_linesButton">
           <property name="text">
            <string>Line Markers</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">_stateButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="_pointsButton">
           <property name="text">
            <string>Point Markers</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">_stateButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="framesLayout">
           <item>
            <widget class="QComboBox" name="_framesComboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Key Frames</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Use the comboBox to select a region, then navigate to a frame of interest and start drawing lines and/or markers.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;When complet move to a new frame, and the lines/markers from the previous will appear green and should be moved where possible.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CGTVideoControls</class>
   <extends>QWidget</extends>
   <header>cgt.gui.cgtvideocontrols</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_markerButtonGroup</sender>
   <signal>buttonClicked(QAbstractButton*)</signal>
   <receiver>ArtifactMarkupWidget</receiver>
   <slot>marking_state_selected(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_regionsComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ArtifactMarkupWidget</receiver>
   <slot>region_selected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_stateButtonGroup</sender>
   <signal>buttonClicked(QAbstractButton*)</signal>
   <receiver>ArtifactMarkupWidget</receiver>
   <slot>mark_type_selected(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_framesComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ArtifactMarkupWidget</receiver>
   <slot>key_frame_selected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="_markerButtonGroup"/>
  <buttongroup name="_stateButtonGroup"/>
 </buttongroups>
</ui>
